uint16[4] pressure_available